problems/objects.js

```diff
@@ -187,15 +260,33 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+let object = countOccurance(str)
+delete object[' ']
```

Comment:
this is a really great way of solving this. My first time seeing it! 

Great work